scripts/simex

```diff
@@ -769,6 +770,51 @@ experiments_print_parser = experiments_subcmds.add_parser('print',
 		parents=[run_selection_parser])
 experiments_print_parser.set_defaults(cmd=do_experiments_print_output)
 
+def do_experiments_kill(args):
+	cfg = extl.base.config_for_dir()
+
+	wanted_status = {}
+	wanted_slurm_jobids = []
+
+	if not (args.submitted and args.started):
+		wanted_status = {Status.SUBMITTED, Status.STARTED}
+	else:
+		if args.submitted:
+			wanted_status.add(Status.SUBMITTED)
+		if args.started:
+			wanted_status.add(Status.STARTED)
+
+	for run in cfg.discover_all_runs():
+		if run.get_status() in wanted_status:
+			slurm_jobid = run.slurm_jobid
+			if slurm_jobid is not None:
+				if not args.f:
+					print("This would kill run {}/{}[{}] with Slurm jobid {}".format(
+						run.experiment.display_name, run.instance.shortname, run.repetition, run.slurm_jobid))
+				else:
+					print("Killing run {}/{}[{}] with Slurm jobid {}".format(
+						run.experiment.display_name, run.instance.shortname, run.repetition, run.slurm_jobid))
+				wanted_slurm_jobids.append(run.slurm_jobid)
+
+	if len(wanted_slurm_jobids) > 0:
+		if not args.f:
+			print("Use '-f' to confirm this action.")
+		else:
+			scancel_cmd = ['scancel']
+			scancel_cmd.extend(wanted_slurm_jobids)
+
+			subprocess.run(scancel_cmd)
+	else:
+		print("No Slurm jobs available")
+
+	cfg.writeback_status_cache()
+
+experiments_kill_parser = experiments_subcmds.add_parser('kill')
+experiments_kill_parser.set_defaults(cmd=do_experiments_kill)
+experiments_kill_parser.add_argument('--submitted', action='store_true', help='Kill pending Slurm jobs')
+experiments_kill_parser.add_argument('--started', action='store_true', help='Kill running Slurm jobs')
```

Comment:
Why do we use a different way to select runs in this function (compared to our usual options as in launch/purge/etc)?

Reply:
It only makes sense to kill experiments that were ``started`` or ``submitted``. But I can allow the usual options and then filter out every status that is not in ``['started', 'submitted']`` so the run selection interface stays consistent

Reply:
Yes, that sounds reasonable.

Reply:
> It only makes sense to kill experiments that were `started` or `submitted`. But I can allow the usual options and then filter out every status that is not in `['started', 'submitted']` so the run selection interface stays consistent

done